documentation/tech-directives/unit-test-instructions.md

```diff
@@ -0,0 +1,43 @@
+# Introduction #
+- The purpose of creating and running Unit test is to check functionality and quality of the code from each part of the software.
+
+# Requirement #
+  - Each **core** functional class are detected by a Unit Test Class.
+  - Within Unit test class, Each core method from the detected class should be attached with **at least two** test Unit Tests     with different given input and expected result.
+  - All the Unit Tests should be **commented** to indicate the detected methods.
+  - To guarantee the functionality, all the Unit tests should be **passed**.
+  - The coverage should be **at least 80 percent** or higher.
+
+# Setup Jest #
+  ### Frontend ###
+  1. Install Jest: run the following command in a terminal: `npm install --save-dev jest@26.6.0`
+  2. Install React test renderer: run the following command in a terminal when you are done #1: `npm install reac-test-            renderer`
```

Comment:
Minor: extra space in `npm install reac-test-            renderer`

Also, "react" is missing a t

Reply:
Corrected. TY!